src/gtk/webview_webkit2.cpp

```diff
@@ -950,7 +950,11 @@ bool wxWebViewWebKit::IsBusy() const
 
 void wxWebViewWebKit::SetEditable(bool enable)
 {
+#if WEBKIT_CHECK_VERSION(2, 8, 0)
     webkit_web_view_set_editable(m_web_view, enable);
+#else
+    g_object_set(m_web_view, "editable", enable ? TRUE : FALSE, NULL);
```

Comment:
Yes, that does make sense, but I guess the other platforms can always enable editing unconditionally, so the API was not made to anticipate failure.